File: ch02_neutronDiffusion/matlab/tref.m
clear
close all

LW = 2;

figure
plot([0,1,0,0],[0,0,1,0],'-o','Color','k','LineWidth',LW,'MarkerFaceColor','k')
xlabel('\eta')
ylabel('\xi')
axis equal
xlim([0,1.1])
ylim(xlim());
set(gca,'FontName','Times New Roman','FontSize',14)
print('../figs/Tref.eps','-depsc2')
close(gcf)

File: ch02_neutronDiffusion/python/tref.py
import numpy as np
import matplotlib.pyplot as plt

LW = 2
FN = 'Times New Roman'
FS = 12
plt.rc('lines', lw=LW)
plt.rc('mathtext', fontset='stix') # not explicitly Times New Roman but a good clone
plt.rc('font', family=FN, size=FS)

plt.figure()
ax = plt.gca()
plt.plot([0, 1, 0, 0], [0, 0, 1, 0], '-ko')
plt.xlabel('$$\\eta$$', usetex=True)
plt.ylabel('$$\\xi$$', usetex=True)
ax.set_aspect('equal', 'box')
plt.ylim(plt.xlim())
plt.savefig('../figs/Tref.pdf', bbox_inches='tight', pad_inches=0)
plt.close()
